### DISCUSSION TITLE
Some rooms are missing in TUMonline

### DISCUSSION THREAD

**Discussion by octycs** [TUM-Dev/NavigaTUM#3085] [Announcements]
**Describe the bug**
As we know from the data compilation, there are a lot of rooms that are in the Roomfinder, but not in TUMonline (note that this is a different issue than #177). I am not sure where we best put this, so I am creating this issue to further discuss.

**To Reproduce**
I've checked as an example the two rooms 1953@0509 and Z951@0509 in the Roomfinder. Both cannot be found in TUMonline by their room numbers (1953 and Z951) or the corresponding TUMonline IDs (0509.01.951 and 0509.02.953). The neighboring rooms exist in TUMonline.

Room Z951 exists physically, but the room number plate has been pasted over. Room 1953 exists and has a room plate:
![image](https://user-images.githubusercontent.com/7429408/184428937-1463aa32-3385-4ca9-aa3c-972186439c17.png)

**Expected behavior**
TUMonline should have all public rooms at TUM.

**Additional context**
The *Stammgelände* seems most affected, but this could also be, because other buildings are less covered in the Roomfinder.

**Comment by CommanderStorm** [comment]
> *This was generated by AI during triage.*

## Triage Notes

**What we've established so far:**

- Reporter identified concrete examples: rooms `1953@0509` and `Z951@0509` exist physically (one with a room plate visible, one with the plate pasted over) but are absent from TUMonline. Both their TUMonline-style IDs (`0509.01.951`, `0509.02.953`) also resolve to nothing.
- These rooms do exist in the (older) Roomfinder data source; NavigaTUM ingests both, so the question is what to surface when the two sources disagree.
- The Stammgelände seems disproportionately affected, consistent with Roomfinder coverage being uneven by area rather than the issue being building-specific.

**What we still need (maintainer decision, not reporter input):**

This is a policy/scope question — what should NavigaTUM do with rooms present in Roomfinder but missing from TUMonline?

- Surface them anyway (with a "from Roomfinder" provenance note) so users can navigate to physically-existing rooms?
- Suppress them to match TUMonline as the source of truth?
- Hybrid: surface only if Roomfinder data is internally consistent (room number plate confirmed, neighbours present)?

Once that policy is decided, the implementation work moves into either the data pipeline (`data/sources/15_patches-rooms_tumonline.yaml` and adjacent) or the merge logic in `data/processors/`. Before then there's nothing concrete for an agent to do.

Re-triage once the policy call is made.
